src/tools/align/components/results/PhyloTree.tsx

```diff
@@ -0,0 +1,375 @@
+// See https://observablehq.com/@mbostock/tree-of-life for original inspiration
+import React, { useEffect, useRef, useMemo, FC } from 'react';
+import { debounce } from 'lodash-es';
+import { cluster, hierarchy, select, max, easeQuadOut } from 'd3';
+import { Message } from 'franklin-sites';
+
+import AlignLabel from './AlignLabel';
+
+import useSize from '../../../../shared/hooks/useSize';
+import useReducedMotion from '../../../../shared/hooks/useReducedMotion';
+
+import phylotree from '../../adapters/phylotree';
+
+import extractAccession from '../../utils/extractAccession';
+import { polarToX, polarToY } from '../../utils/trigonometry';
+import pathMaker from '../../utils/pathMaker';
+import customLayout, { CustomHierarchyNode } from '../../utils/customLayout';
+
+import { PhyloTreeNode } from '../../types/alignResults';
+
+import './styles/PhyloTree.scss';
+
+const DURATION = 500;
+const DEBOUNCE_DELAY = 250;
+// keep that in case it's needed at a later point as it is now already included
+// in all the calculations thorugh this file anyway
+const MARGIN = 0;
+
+interface Redraw {
+  ({
+    width,
+    showDistance,
+    alignLabels,
+    circularLayout,
+  }: {
+    width: number;
+    showDistance: boolean;
+    alignLabels: boolean;
+    circularLayout: boolean;
+  }): void;
+}
+
+interface Cancelable {
+  cancel(): void;
+  flush(): void;
+}
+
+type Props = {
+  newick?: string;
+  showDistance: boolean;
+  alignLabels: boolean;
+  circularLayout: boolean;
+};
+
+const PhyloTree: FC<Props> = ({
+  newick,
+  showDistance,
+  alignLabels,
+  circularLayout,
+}) => {
+  const svgRef = useRef<SVGSVGElement>(null);
+  const redrawRef = useRef<Redraw & Cancelable>();
+
+  const [size] = useSize<SVGSVGElement>(svgRef);
+  const reducedMotion = useReducedMotion();
```

Comment:
Well, technically, it could change over time. In the reaaaaally improbable case that someone goes and change their system setting to while their navigating on the website...
Not sure if I overdid it here and should have just assumed it would stay the same.

I was also thinking that this could eventually be a hook linked to other data from the browser (site-specific user settings, battery level, processing power of the machine). Hence why I made it in a hook, because some of them can change over time during the browsing session

Reply:
Makes sense to keep as a hook then as you have plans to expand to more dynamic data.